js/classes/entities/midground/enemies/village/snake.js

```diff
@@ -132,7 +132,7 @@ class Snake {
                 this.action = "idle";
                 if (Date.now() - Snake.ATTACK_COOLDOWN * 1000 > this.lastAttack) {
```

Comment:
I like this, maybe we can make a constant in our project for 1000 as a conversion constant. Not necessary, just a thought.